locksmith/src/utils/pricing.ts

```diff
@@ -122,7 +122,18 @@ export const getCreditCardProcessingFee = (
 }
 
 // Fee denominated in cents
-export const getUnlockServiceFee = (cost: number) => {
+export const getUnlockServiceFee = (
+  cost: number,
+  options?: KeyPricingOptions
+) => {
+  if (
+    options?.lockAddress.toLowerCase() ===
```

Comment:
Add .trim() too

Reply:
Actually I will clean up all these `.toLowerCaser().trim()` and use the ormalizer in another PR

Reply:
https://github.com/unlock-protocol/unlock/pull/12515